projects/scannet_offline_eval/semantic_map_3d.ipynb

```diff
@@ -206,7 +205,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "encoded_classes = [encoder.encode_text]"
+    "# encoded_classes = [encoder.encode_text]"
```

Comment:
should we remove this?

Reply:
Yes this notebook is sort of my scratch pad. Maybe we should come up with a standard place for notebooks (e.g. `examples/`)

Reply:
Yeah, examples/ or notebooks/ sounds good to me

Reply:
especially if it only depends on core code